sass/dimensions.scss

```diff
@@ -0,0 +1,5 @@
+$num-of-columns: 12;
```

Comment:
I guess this is easy enough to change, but is 12 columns enough for the kind of layouts we do?